packages/now-next/src/index.ts

```diff
@@ -456,21 +508,31 @@ exports.prepareCache = async ({ workPath, entrypoint }) => {
   };
 };
 
-exports.subscribe = async ({ entrypoint, files }) => {
-  const entryDirectory = path.dirname(entrypoint);
-  const pageFiles = includeOnlyEntryDirectory(
+exports.shouldServe = async ({ entrypoint, files, requestPath }: {entrypoint: string, files: Files, requestPath: string}) => {
```

Comment:
```suggestion
export const shouldServe = async ({ entrypoint, files, requestPath }: {entrypoint: string, files: Files, requestPath: string}) => {
```

I can't remember if `ShouldServeOptions` is merged yet but it soon will be.